src/components/Progress/Progress.scss

```diff
@@ -0,0 +1,3 @@
+.progress-percentage {
```

Comment:
should be named as `asab-progress-percentage`, that it is not confused with other progresses